src/Core/Data/PostedItem.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.Data
+{
+	public class PostedItem
+	{
+
+		public UserProfile Author { get; set; }
+
+		public DateTime DateCreated { get; set; }
+
+		public DateTime? LastEdit { get; set; }
+
+		public string Title { get; set; }
+
```

Comment:
Should there be an ID property?  The "Inheritance with EF Code First: Part 2 – Table per Type (TPT)" article seemed to indicate that an ID is required for the pattern.